internal/app/configs/configs.go

```diff
@@ -1,42 +1,86 @@
 package configs
 
 import (
+	"encoding/json"
 	"flag"
+	"log"
 	"os"
+	"strconv"
 )
 
 // Application configs
 type Config struct {
-	ServerAddress        string
-	ShortenedURLBaseAddr string
-	FileStoragePath      string
-	DatabaseDSN          string
+	ServerAddress   string `json:"server_address,omitempty"`
+	BaseURL         string `json:"base_url,omitempty"`
+	FileStoragePath string `json:"file_storage_path,omitempty"`
+	DatabaseDSN     string `json:"database_dsn,omitempty"`
+	EnableHTTPS     bool   `json:"enable_https"`
 }
 
-// Parse configs from flag variables. Environment variables override flag variables
+// Parse configs
 func Parse() Config {
-	config := Config{}
-	flag.StringVar(&config.ServerAddress, "a", "localhost:8080", "server's address")
-	flag.StringVar(
-		&config.ShortenedURLBaseAddr,
-		"b", "http://localhost:8080",
-		"base address of the resulting shortened URL")
-	flag.StringVar(&config.FileStoragePath, "f", "", "file storage path")
-	flag.StringVar(&config.DatabaseDSN, "d", "", "database URL")
+	var (
+		flagServerAddress   string
```

Comment:
```suggestion
		flagServerAddress  = flag.String("a", "", "server's address")
```
Можно сразу значения записать